client/my-sites/stats/hooks/use-should-gate-stats.ts

```diff
@@ -165,12 +171,6 @@ export const shouldGateStats = ( state: object, siteId: number | null, statType:
 		return [ ...jetpackStatsAdvancedPaywall ].includes( statType );
 	}
 
-	// Gate advanced stats for non-Jetpack sites unless they have a Jetpack Stats commercial purchase.
-	// Dotcom sites are not able to see these modules yet, so the line wouldn't apply to them.
-	if ( jetpackStatsAdvancedPaywall.includes( statType ) ) {
-		return ! supportStatsCommercialUse;
-	}
-
```

Comment:
I’m removing this because it was intended for non-Jetpack sites. Gating checks for Jetpack sites already occurs in the preceding block.

Reply:
That's it!